frontend/src/modules/modals/MultiStepWizard.tsx

```diff
@@ -865,20 +805,19 @@ const MultiStepWizard = (): JSX.Element => {
               </Grid>
             </Grid>
             <Grid item className={classes.marginContent}>
-              {multiParams.map((param: Data) => {
+              {multiParams.map((param: InternalState) => {
                 if (param.isSetvalEnabled) {
-                  if (param.setValue !== param.setValueActual) {
+                  if (param.ValueDraft !== param.ValueCurrent) {
                     return (
-                      <Typography variant="subtitle1">{`Keep ${param.label} to ${param.setValueActual} ${param.units}?`}</Typography>
+                      <Typography variant="subtitle1">{`Keep ${param.label} to ${param.ValueCurrent} ${param.units}?`}</Typography>
                     );
                   }
                 } else if (
-                  param.alarmValues.length &&
-                  (param.alarmValues[0] !== param.alarmValuesActual[0] ||
-                    param.alarmValues[1] !== param.alarmValuesActual[1])
+                  param.alarmValues.lower !== param.alarmValuesActual.lower ||
+                  param.alarmValues.upper !== param.alarmValuesActual.upper
                 ) {
                   return (
-                    <Typography variant="subtitle1">{`Keep ${param.label} alarm range to ${param.alarmValuesActual[0]} - ${param.alarmValuesActual[1]}?`}</Typography>
+                    <Typography variant="subtitle1">{`Keep ${param.label} alarm range to ${param.alarmValuesActual.lower} - ${param.alarmValuesActual.upper}?`}</Typography>
```

Comment:
Let's see what usability is like if we also display the change we're discarding here, too: e.g. Keep HR alarm limits at [0 BPM - 200 BPM], instead of changing to [50 BPM - 100 BPM]?